include/swift/AST/DiagnosticsSema.def

```diff
@@ -825,6 +825,11 @@ ERROR(argument_out_of_order_unnamed_unnamed,none,
       "unnamed argument #%0 must precede unnamed argument #%1",
       (unsigned, unsigned))
 
+ERROR(member_shadows_global_function,none,
+      "class member %0 shadows %1 %0 in %2 '%3'"
```

Comment:
You've hard-coded `class member`, but couldn't the same problem happen for a struct or enum, or perhaps even a protocol or protocol extension?